todo_app/trello_api_calls.py

```diff
@@ -58,4 +66,25 @@ def change_card_status(card_id,to_state="Done"):
     list_id = get_list_id(to_state)
     url = api_prefix + '1/cards/' + card_id
     listdata = {"idList" : list_id}
-    api_response = requests.put(url,params=payload, data = listdata)
+    requests.put(url,params=payload, data = listdata)
+
+# Written solely for the purpose of testing. 
+def create_trello_board():
+    payload = {
+        'key': os.environ.get('TRELLO_KEY'),
+        'token': os.environ.get('TRELLO_TOKEN'),
+        'name': os.environ.get('TRELLO_BOARD_NAME_CREATE')
```

Comment:
This environment variable doesn't seem to be documented anywhere - we should add it to the README/`.env.template` files so that future devs would know this is required for the tests

Reply:
Agreed, added to `.env.template` and `.env.test`